openslides/core/static/js/core/site.js

```diff
@@ -526,32 +526,79 @@ angular.module('OpenSlidesApp.core.site', [
     '$scope',
     '$state',
     function ($scope, $state) {
-        $scope.search = function(query) {
-            $scope.query = '';
+        $scope.search = function() {
```

Comment:
please clear the search bar after searching - keep it empty for entering new search string.